benchmark/bm_entangling_layers.py

```diff
@@ -56,5 +56,5 @@ def benchmark(self, n=3):
 
         qnode = bu.create_qnode(circuit, self.device, mutable=False)
         qnode(init_weights, features=features)
-        qnode.jacobian(init_weights, {"features": features})
+        qnode.jacobian((init_weights,), {"features": features})
```

Comment:
I decided to sneak in a quick bugfix to another benchmark.

Reply:
@trbromley noted this as well! He also noted the need to mark it `mutable` in order to get it working?

Reply:
That's a different thing, a workaround to a bug in the current `master` branch. The bug is fixed by #510.